.github/workflows/pythonbuild.yml

```diff
@@ -152,6 +153,100 @@ jobs:
           fail_ci_if_error: false
           files: coverage.xml
 
+  build-without-grpcio:
+    needs:
+      - detect-python-versions
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ${{fromJson(needs.detect-python-versions.outputs.python-versions)}}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Cache pip
+        uses: actions/cache@v3
+        with:
+          # This path is specific to Ubuntu
+          path: ~/.cache/pip
+          # Look to see if there is a cache hit for the corresponding requirements files
+          key: ${{ format('{0}-pip-{1}', runner.os, hashFiles('dev-requirements.in', 'requirements.in')) }}
+      - name: Build and Install Rust client
+        working-directory: flyrs
+        run: |
+          pip install maturin
+          maturin build --release --out dist --interpreter python ${{ env.python_versions }}
+          pip install $(find ./dist -name '*.whl' | head -n 1)
+      - name: Install dependencies
+        run: |
+          make setup
```

Comment:
Nit: could we use uv here instead? here is an example. https://github.com/flyteorg/flytekit/blob/1ca3d91876002534f656688c327f7411ad3f1f89/.github/workflows/pythonbuild.yml#L60-L63

Reply:
No problem. Thank you for providing examples and it's much faster now.